src/main/kotlin/com/jaredsburrows/license/LicenseReportTask.kt

```diff
@@ -164,6 +164,11 @@ internal open class LicenseReportTask : DefaultTask() { // tasks can't be final
       .lenientConfiguration
       .artifacts.forEach { resolvedArtifact ->
 
+        // Skip parsing xml for non-pom files
+        if (resolvedArtifact.type != "pom") {
+          return@forEach
+        }
+
         val pomFile = resolvedArtifact.file
         val node = xmlParser.parse(pomFile)
```

Comment:
with the android gradle plugin **< 7.1.0** following dependencies are getting resolved within our example project:
``` 
resolvedArtifact.id: vectordrawable-animated-1.1.0.pom (androidx.vectordrawable:vectordrawable-animated:1.1.0)
resolvedArtifact.type: pom
resolvedArtifact.file: /Users/flobetz/.gradle/caches/modules-2/files-2.1/androidx.vectordrawable/vectordrawable-animated/1.1.0/2c40e16427f4e94ea471526868ba5ac4bd13de6d/vectordrawable-animated-1.1.0.pom
```
with the android gradle plugin **>= 7.1.0** following dependencies are getting resolved within our example project:
``` 
resolvedArtifact.id: vectordrawable-animated-1.1.0.pom (androidx.vectordrawable:vectordrawable-animated:1.1.0)
resolvedArtifact.type: pom
resolvedArtifact.file: /Users/flobetz/.gradle/caches/modules-2/files-2.1/androidx.vectordrawable/vectordrawable-animated/1.1.0/2c40e16427f4e94ea471526868ba5ac4bd13de6d/vectordrawable-animated-1.1.0.pom
-----
resolvedArtifact.id: vectordrawable-animated-1.1.0.aar (androidx.vectordrawable:vectordrawable-animated:1.1.0)
resolvedArtifact.type: aar
resolvedArtifact.file: /Users/flobetz/.gradle/caches/modules-2/files-2.1/androidx.vectordrawable/vectordrawable-animated/1.1.0/fcda1161354501471c30a4e077af6b5c4d4eddc6/vectordrawable-animated-1.1.0.aar
```

The gradle-license-plugin will obvously fail when trying to parse a .aar file which is currently done at L173
@jaredsburrows can you have a look at this please?